.changeset/shaggy-keys-turn.md

```diff
@@ -0,0 +1,63 @@
+---
+'astro': major
+'@astrojs/markdown-remark': major
+'@astrojs/mdx': minor
+---
+
+Refine Markdown and MDX configuration options for ease-of-use.
+
+## Markdown
```

Comment:
We might want to start headers with `####` since the changelog already occupies h1, h2, and h3.